pymatgen/analysis/ferroelectricity/ZVAL.yaml

```diff
@@ -0,0 +1,305 @@
+VASP_PBE_ZVAL:
```

Comment:
Yes, please modify Outcar to get the ZVAL. VASP will always issue new psp and I don't think we should be trying to trying to maintain a moving target.